python/nav/web/auth.py

```diff
@@ -260,42 +260,54 @@ def login_remote_user(request):
     return None
 
 
+def ensure_account(request):
```

Comment:
Again, since this is a callout function, I would prefer that is defined below the "meat" of the module (screw Pascal!) :-)

Kudos for factoring it out, it makes the whole thing more readable.

Reply:
Also fixed in a fixup-patch